applications/crossbar/doc/search.md

```diff
@@ -39,3 +47,35 @@ The Search API allows queries on databases.
     }
 
 
+### _GET_ - Multi Search
+
+    t = document type
+    view_name = value
+
+
+    curl -v -X GET -H "X-Auth-Token: {AUTH_TOKEN}" 'http://localhost:8000/v2/search?t=account&name=test&realm=test'
+
+
+```
+{
+  "data": {
+    "realm": [],
```

Comment:
Why does the query above have `name==test&realm=test` but the response has empty arrays for `name` and `realm`? The code doesn't appear to put the results of the search into the right buckets.